src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1427,9 +1428,10 @@ public function testInheritEnvDisabled()
      *
      * @return Process
      */
-    private function getProcess($commandline, $cwd = null, array $env = null, $input = null, $timeout = 60, array $options = array())
+    private function getProcess($commandline, $cwd = null, array $env = null, $input = null, $timeout = 60)
     {
-        $process = new Process($commandline, $cwd, $env, $input, $timeout, $options);
+        $process = new Process($commandline, $cwd, $env, $input, $timeout);
+        $process->inheritEnvironmentVariables(true, false);
```

Comment:
Well, we never asked them to use our internal silencing arguments before, because using them makes it far too easy to call the method without warning *without* checking for it to be safe, and because these were meant to be internal.
Btw, your second argument is not even part of the method signature, meaning that their IDEs will complain about passing unknown arguments.
I don't think this is the best upgrade path.

On a side note, if we really want to keep this upgrade path, we should give them the right update path from the start IMO.